### ISSUE TITLE
On running the command: python manage.py create_custom_user_app    Following error is generated -    KeyError: 'skip checks' .    /django_use_email_as_username/management/commands/create_custom_user_app.py", options.pop("skip_checks") KeyError: 'skip_checks'

### ISSUE THREAD

**Issue by ankit19s** [jmfederico/django-use-email-as-username#5]
* Django use Email as Username version:
* Django version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by jmfederico** [comment]
Which version are you using?

**Comment by blindpenguin** [comment]
Same here with Django 3.0.3 on Windows 10. App version is 1.1.0.

**Comment by jmfederico** [comment]
Thanks for reporting.

Fixed in [v1.1.1](https://pypi.org/pypi/django-use-email-as-username/1.1.1/).

Please reopen if you still find the issue.
